submission/proxy.go

```diff
@@ -75,7 +76,12 @@ func ASN1MarshalSCTs(scts []*AssignedSCT) ([]byte, error) {
 	if err != nil {
 		return nil, err
 	}
-	encoded, err := asn1.Marshal(sctList)
+	encdSCTList, err := tls.Marshal(*sctList)
+	if err != nil {
+		fmt.Printf("Err on tls: %v\n", err)
```

Comment:
This should probably be a log.Errorf() rather than printing it, or remove it.

Reply:
Removed.